localflavor/by/forms.py

```diff
@@ -0,0 +1,102 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from .by_regions import BY_REGIONS_CHOICES
+
+
+class BaseKwargsUpdatedField:
+    """
+    Abstract base class made to conform the DRY principle.
+
+    The initial_options is overrided by the dict in the subclasses and then
+    automatically passed to kwargs in the FormField.__init__() method.
+    """
+
+    initial_options = None
+
+    def __init__(self, *args, **kwargs):
+        kwargs.update(self.initial_options)
+        super().__init__(*args, **kwargs)
+
+
+class UpperValueMixin:
+    """
+    Mixin made to conform the DRY principle.
+
+    Overrides the to_python methods to return the value.upper()
+    """
+
+    def to_python(self, value):
+        value = super().to_python(value)
+        if value is None:
+            return value
```

Comment:
I don't think it's a good idea to use `None` as the default for empty value. `'empty_value': None` should be remove from the `initial_options` and this code should be updated like this: 
```suggestion
        if value in self.empty_values:
            return self.empty_value
```
This will let users set `None` if they prefer this. Otherwise an empty string will be used.